src/components/BlockCapabilityBlockDocumentInput.vue

```diff
@@ -1,16 +1,32 @@
 <template>
-  <div class="block-capability-block-document-input">
-    <BlockTypeLogo v-if="blockDocument" :block-type="blockDocument.blockType" />
-
-    <BlockDocumentsSelect v-model:selected="internalModalValue" class="block-capability-block-document-input__select" :block-documents="blockDocuments" required />
+  <div class="block-capability-block-document-input" :class="classes" :style="styles" v-bind="listeners">
```

Comment:
```suggestion
  <div class="block-capability-block-document-input" :class="classes" :style="styles" v-bind="listeners">
```
Given that this is the root element I'm surprised to see this is necessary; is there something about the `useAttrsStylesClassesAndListeners` component that doesn't allow you just to use/bind the `attrs` down below?

Reply:
We use this when we basically want to pass props/attrs but not styles, classes, or listeners. Specifically used here so `state` can be bound to the select.

Reply:
I missed an extremely critical line of your PR, sorry lol:
```
    inheritAttrs: false,
```

Reply:
lol no worries!